lib/tasks/app_setup.rake

```diff
@@ -0,0 +1,20 @@
+namespace :app do
+  desc "Prepare Application on Boot Up"
+  task setup: :environment do
+    puts "\n== Preparing Elasticsearch =="
+    Rake::Task["search:setup"].execute
+
+    puts "\n== Preparing database =="
+    begin
+      Rake::Task["db:migrate"].execute
+    rescue ActiveRecord::NoDatabaseError
```

Comment:
This should also work better than the way we were doing it before. I have been caught off guard a couple of times when my database was reset due to a random migration that had nothing to do with a database missing.

Reply:
Cool! Fortunately Rails 6 adds `db:prepare` which does just that. I'll update the branch when we merge this